src/index.ts

```diff
@@ -147,7 +147,7 @@ async function main() {
   } else {
     const transport = new StdioServerTransport();
     await server.connect(transport);
-    console.log("MCP server running with stdio transport");
+    console.error("MCP server running with stdio transport");
```

Comment:
What is the best practice?
I think maybe we should remove the log completely.

This specific log entry is for human, which not intended to run the stdio mode just from commandline without arguments.